services/graph/pkg/service/v0/utils.go

```diff
@@ -425,3 +433,104 @@ func cs3ReceivedShareToLibreGraphPermissions(ctx context.Context, logger *log.Lo
 
 	return permission, nil
 }
+
+func (g Graph) applySpaceTemplate(gwc gateway.GatewayAPIClient, root *storageprovider.ResourceId, template string) (*v1beta1.Opaque, error) {
+	var fsys fs.ReadDirFS
+
+	switch template {
```

Comment:
It is just `if else` 
re we going to extend this in the future?

Reply:
For now it has only one option, yes. So could change it to a if clause if you think that is simpler. However we are planning to extend the templates with cutom options later. 

Not sure what is better for now. Would you prefer an if clause?

Reply:
Let's keep it like as is